resources/views/auth/info.blade.php

```diff
@@ -0,0 +1,124 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 col-md-offset-1">
+                <div class="panel panel-default">
+                    <div class="panel-body">
```

Comment:
We use the "card" class instead of the "panel"

Reply:
fixed